konflux-configs/base/application/model-validation-operator/base/kustomization.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - application.yaml
+  - component.yaml
+  - imagerepository.yaml
+  - ec-group.yaml
+  - monorepo.yaml
+
+components:
+  - ../../base/application
+
+patches:
+  - path: patch/ec.yaml
+    target:
+      kind: IntegrationTestScenario
+      name: model-validation-operator-enterprise-contract
+
+  - patch: |-
```

Comment:
Instead of deleting the promote-to-candidate releasePlan for model transparency and its operator, can we add it? I think its currently missing from the cluster, I also think they should be pushed to there own files using the
file-name param e.g model-transparency.json, and model-validation-operator.json